scenarioo-client/bower.json

```diff
@@ -2,29 +2,16 @@
     "name": "scenarioo-client",
     "version": "0.0.0",
     "dependencies": {
-        "angular": "1.5.2",
-        "angular-resource": "1.5.2",
-        "angular-sanitize": "1.5.2",
-        "angular-route": "1.5.2",
-        "angular-i18n": "1.5.2",
-        "angular-local-storage": "~0.2.5",
-        "bootstrap": "~3.3.2",
-        "angular-bootstrap": "~1.2.2",
         "font-awesome": "~3.2.1",
-        "twigs": "0.1.3.1",
         "svg.js": "https://github.com/svgdotjs/svg.js.git#2.0.4",
         "svg.draggable.js": "https://github.com/svgdotjs/svg.draggable.js.git#2.0.0",
         "svg.select.js": "https://github.com/parabolika66/svg.select.js.git#~1.0.8",
         "svg.resize.js": "https://github.com/parabolika66/svg.resize.js.git#1.1.3",
-        "svg-pan-zoom": "~3.2.0",
         "angular-unsavedChanges": "~0.1.1"
     },
     "devDependencies": {
-        "angular-mocks": "1.5.2",
-        "angular-scenario": "1.5.2"
     },
     "resolutions": {
-        "svg.js": "2.0.4",
-        "angular": "1.5.2"
+        "svg.js": "2.0.4"
```

Comment:
because we have some old weird dependencies, which do not exist in the same version on npm. For svg only this exact version works. It's crazy but we've got a huge technical debt there. Also there's an issue with font-awesome v3. We would need to upgrade it to v4.

Reply:
It's there but there will be some jekyll error, very strange actually.

Reply:
Jekyll error?? Wtf? Okeee